torch/fx/experimental/proxy_tensor.py

```diff
@@ -1128,6 +1133,24 @@ def _should_save_arg_kwarg_vals(target: Any) -> bool:
         ),
     ):
         return True
+    if args_kwargs is not None and (
+        target is torch.ops.higher_order.auto_functionalized
+        or target is torch.ops.higher_order.auto_functionalized_v2
+    ):
+        args = args_kwargs[0]
+        if isinstance(args[0], torch._ops.OpOverload):
+            return _should_save_arg_kwarg_vals(args[0], None)
+    if target is torch.ops.higher_order.with_effects:
```

Comment:
I guess this is the target ? Also, if not `if isinstance(args[0], torch._ops.OpOverload):` should we return False instead of falling to other cases ?